src/components/Header.jsx

```diff
@@ -33,7 +33,7 @@ export default class Header extends BaseComponent {
           <div className="header__title">
             {/* TODO: change link to proper Gazelle icon uploaded to server*/}
             <Link to="/" className="header__title__content">
-              <img src="https://thegazelle.s3.amazonaws.com/gazelle/2016/02/header-logo.png" alt="logo" />
+              <img src="./static/header-logo-transparent.png" alt="logo" />
```

Comment:
This image is going to have to be uploaded through the editor tools so we can load it in with our CDN. Right now the image isn't loading in.

Reply:
Could I e-mail you the image so you can upload it / can we do this together on Sunday? I still don't have it set up so I can upload things myself, I think.

Reply:
Yeah either load it with CDN (which is probably the best, yep) or add it to the static folder (but probably just as good doing it with S3 I agree).